src/app/helpers/whitelistApis.js

```diff
@@ -996,15 +996,15 @@ const API_LIST = {
         ROLE.BOOK_CREATOR
       ]
     },
-    '/action/dialcode/v1/reserve/:do_id': {
```

Comment:
The end-point we should not change. If it is changes, front-end also requires changes.